src/rulesets/LinearAlgebra/dense.jl

```diff
@@ -129,7 +129,7 @@ frule((_, Δx), ::typeof(det), x::Number) = (det(x), Δx)
 function rrule(::typeof(det), x::Union{Number, AbstractMatrix})
     Ω = det(x)
     function det_pullback(ΔΩ)
-        ∂x = x isa Number ? ΔΩ : Ω * ΔΩ * inv(x)'
+        ∂x = x isa Number ? ΔΩ : conj(Ω) * ΔΩ * inv(x)'
```

Comment:
Perhaps we should explicitly multiple the two scalars together. Also, this computes the product in the correct order for numbers for which multiplication isn't commutative (though probably the rule is not even correct for such numbers anyways):
```suggestion
        ∂x = x isa Number ? ΔΩ : inv(x)' * dot(Ω, ΔΩ)
```

Reply:
Sure, that looks good to me!